go.spec

```diff
@@ -53,20 +53,39 @@ any older distro-provided versions.
 %build
 
 %install
+%if (0%{?suse_version} > 0)
+%{__mkdir_p} %{buildroot}/%{_exec_prefix}/bin
+%{__mkdir_p} %{buildroot}/%{_exec_prefix}/lib64/go/%{_go_rel}
+cp -a bin %{buildroot}/%{_exec_prefix}/lib64/go/%{_go_rel}/
+cp -a pkg %{buildroot}/%{_exec_prefix}/lib64/go/%{_go_rel}/
+cp -a src %{buildroot}/%{_exec_prefix}/lib64/go/%{_go_rel}/
+ln -s %{_exec_prefix}/lib64/go/%{_go_rel}/bin/go %{buildroot}/%{_exec_prefix}/bin/go
+ln -s %{_exec_prefix}/lib64/go/%{_go_rel}/bin/gofmt %{buildroot}/%{_exec_prefix}/bin/gofmt
+%else
 %{__mkdir_p} %{buildroot}/%{_exec_prefix}
 cp -a bin %{buildroot}/%{_exec_prefix}
 cp -a pkg %{buildroot}/%{_exec_prefix}
 cp -a src %{buildroot}/%{_exec_prefix}
+%endif
 
 %files
+%if (0%{?suse_version} > 0)
+%{_bindir}/*
+%{_exec_prefix}/lib64/go/%{_go_rel}/bin/*
+%{_exec_prefix}/lib64/go/%{_go_rel}/pkg/*
+%{_exec_prefix}/lib64/go/%{_go_rel}/src/*
+%else
 %{_bindir}/*
 %{_exec_prefix}/pkg/*
 %{_exec_prefix}/src/*
+%endif
+
 %doc
 
 %changelog
 * Wed Sep 13 2023 Lei Huang <lei.huang@intel.com> - 1.21.1-1
```

Comment:
Let's update the datestamp here on your next push also please.

Reply:
Just updated the date stamp. Thank you!